server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -3821,7 +3821,7 @@ private void checkIfHostNameUniqueInNtwkDomain(String hostName, List<? extends N
                 // * verify that there are no duplicates
                 if (hostNames.contains(hostName)) {
                     throw new InvalidParameterValueException("The vm with hostName " + hostName + " already exists in the network domain: " + ntwkDomain.getKey() + "; network="
-                            + _networkModel.getNetwork(ntwkId));
+                            + _networkModel.getNetwork(ntwkId).getName());
```

Comment:
How about fixing/extending the toString of the returned network object (the VO?)

Reply:
```suggestion
                            + (_networkModel.getNetwork(ntwkId) != null) _networkModel.getNetwork(ntwkId).getName()) : "<unknown>";
```